src/metaAnalysis.ts

```diff
@@ -2,6 +2,11 @@ import { AirtableRecord } from "./types"
 
 const Z_SCORE: number = 1.96;
 
+export enum aggregationFactor {
```

Comment:
might be more consistent to put this in types.ts. not sure that's necessary tho, up to you